image-filters/image-filters.markdown

```diff
@@ -3,148 +3,148 @@ author: Cate Huston
 
 _Cate Huston is a developer and entrepreneur focused on mobile. She’s lived and worked in the UK, Australia, Canada, China and the United States, as an engineer at Google, an Extreme Blue intern at IBM, and a ski instructor. Cate speaks internationally on mobile development and her writing has been published on sites as varied as Lifehacker, The Daily Beast, The Eloquent Woman and Model View Culture. She co-curates Technically Speaking, blogs at Accidentally in Code and is [\@catehstn](https://twitter.com/catehstn) on Twitter._
 
-## A Story of a Brilliant Idea That Wasn’t All That Brilliant
-
-In Chinese art, there is often a series of four paintings showing the same
-place in different seasons. Color - the cool whites of winter, pale hues of
-spring, lush greens of summer, and red and yellows of fall is the
-differentiation. Sometime around 2011, I had what I thought was a brilliant
-idea. I wanted to be able to visualize a photo series, as a series of colors. I
+## A Brilliant Idea (That Wasn’t All That Brilliant)
+
+Imagine a series of four paintings showing the same
```

Comment:
sounds good - I saw these constantly in China, but also couldn't find any kind of reference to them.